README.md

```diff
@@ -57,15 +62,46 @@ cordova build
 
 ## Client-side CDC CDSi validator
 
-Limited scope, CQL engine based client-side validation script that executes the CDSi supporting data guidelines.
+The SMART Health Card verification process uses a limited-scope, CQL engine-based, client-side validation script that executes the Clinical Decision Support for Immunization (CDSi) Supporting Data guidelines.
+
+### CDSi Supporting Data
+
+The CDSi Supporting Data guidelines are used to capture recommendations from the Advisory Committee on Immunization Practices (ACIP) for development and maintenance of Clinical Decision Support (CDS) engines. The Supporting Data describe, by antigen, the necessary attributes and specific values to support evaluation of vaccine series when implementing ACIP recommendations.
+
+See the [CDSi Logic Specification and Supporting Data Guidelines](https://www.cdc.gov/vaccines/programs/iis/cdsi.html#logic) for more information.
 
-Currently only validates each non-Risk type series based on allowable series age to start, dose absMinAge, allowable vaccine codes (CVX only) and begin/end age, inadvertent vaccine codes (CVX only), absolute minimum preferable interval from previous dose (including effective and cessation dates), and absolute minimum allowable interval from previous dose.
+### Using the CDSi Supporting Data
 
-Does not support forecasting or any other validation, including conditional doses, recurring or seasonal doses, immunity or contradictions.
+The web verifier utilizes the CDSi Supporting Data guidelines to easily stay up-to-date with the evolving ACIP recommendations. The CDSi Supporting Data are used as input to the [CQL](https://ecqi.healthit.gov/cql) engine. If the scanned SMART Health Card contains COVID-19 vaccination credentials, the SMART Health Card data will undergo CQL validation based on the CDSi Supporting Data. This validation determines whether the vaccine series contained in the SMART Health Card is valid - the results of this validation appear on the display page after the SMART Health Card QR code is scanned.
```

Comment:
**"If the scanned SMART Health Card contains a COVID-19 vaccination credential,"**

We may want to be more specific about when we do / do not run our validation engine. I think it would help to say that we run CQL validation if the `#immunization` and `#covid19` VC Types are present (https://spec.smarthealth.cards/vocabulary/), and if they are not present then validation is skipped.